src/app/content/search/hooks/index.ts

```diff
@@ -63,24 +62,13 @@ export const receiveSearchHook: ActionHookBody<typeof receiveSearchResults> = (s
     ? {result: searchResultHit, highlight: meta.searchScrollTarget.index}
     : null;
 
-  if (
-    selectedResult
-    // We are clearing selected result when requesting a new search so in the theory this should never happen
-    && (isEqual(select.selectedResult(state), selectedResult)
-      // selectedResult bookId data is different than current book id
-      || book.id !== getIndexData(selectedResult.result.index).bookId)
-  ) {
-    return;
-  }
-
   if (selectedResult) {
```

Comment:
@TomWoodward I added you as a reviewer so you could sanity check some decisions I made, such as removing the code above and below.